src/api/app/views/webui2/webui/staging/workflows/_problems_item.html.haml

```diff
@@ -0,0 +1,16 @@
+- project.problems[index].each do |name, states|
+  %li.list-group-item.table-list-group-item
+    = link_to("#{name}:", package_show_path(package: name, project: project))
+    - states.each do |state, values|
+      = state
+      = surround '(', ')' do
+        - values.each do |value|
```

Comment:
@Ana06 I checked and it's indeed the case that the backend returns the architectures in the order they are present in _meta. So the order is by no means random